api/models/reservation.py

```diff
@@ -0,0 +1,31 @@
+"""Payment model."""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from .base import Base
+from .order import Order
+from .workshopPrice import WorkshopPrice
+
+
+class Reservation(Base):
+    """Stores workshop reservations."""
+
+    workshop_price = models.ForeignKey(
+        WorkshopPrice,
+        verbose_name=_("Workshop price"),
+    )
+    order = models.ForeignKey(
+        Order,
+        verbose_name=_("Order"),
+    )
+    ends_at = models.DateTimeField(
```

Comment:
We should stick to one convention on attribute names. Underscore case will cause us trouble on frontend because everything is CamelCase there. Of course it is possible to convert them somehow, but let's keep it simple and clean / there already is `createdAt`, `updatedAt` and now we have `ends_at`. Is there possiblity to add some linting rule on this in Python?